plugin/pkg/admission/limitranger/admission.go

```diff
@@ -249,6 +249,42 @@ func mergePodResourceRequirements(pod *api.Pod, defaultRequirements *api.Resourc
 		}
 	}
 
+	for i := range pod.Spec.InitContainers {
+		container := &pod.Spec.InitContainers[i]
```

Comment:
Added a review comment.

On Fri, May 6, 2016 at 12:48 PM, Derek Carr <notifications@github.com>
wrote:

> In plugin/pkg/admission/limitranger/admission.go
> <https://github.com/kubernetes/kubernetes/pull/23567#discussion_r62356528>
> :
>
> > @@ -249,6 +249,42 @@ func mergePodResourceRequirements(pod *api.Pod, defaultRequirements *api.Resourc
> >  		}
> >  	}
> >
> > +	for i := range pod.Spec.InitContainers {
> > +		container := &pod.Spec.InitContainers[i]
>
> are you responding to the same message? this is doing a merge, not a sum
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/23567/files/805cb1e4b870ef2f22562afd5ecd559c82909659..c3dad6ac86987ff44c2eb12a4707e7636803e48d#r62356528>
>